database/sampleDataMaker.js

```diff
@@ -64,4 +80,52 @@ createList();
 
 module.exports.createDataObj = createDataObj;
 
-//node --max-old-space-size=8192
\ No newline at end of file
+//node --max-old-space-size=8192
```

Comment:
You may consider just removing all the commented code

Reply:
Thank you for your comment Ike